src/common/camera_control.c

```diff
@@ -995,7 +995,7 @@ int _camctl_recursive_get_previews(const dt_camctl_t *c, dt_camera_preview_flags
             else if(!strncmp(c->active_camera->port, "disk:", 5))
             {
               char fullpath[PATH_MAX] = { 0 };
-              snprintf(fullpath, sizeof(fullpath), "%s/%s/%s", c->active_camera->port + 5, path, filename);
+              snprintf(fullpath, sizeof(fullpath), "%s%s/%s", c->active_camera->port + 5, path, filename);
```

Comment:
I'd keep the `/`, unless you can find documentation that guarantees that gphoto's string is always ending in a `/`.